docs/changes.rst

```diff
@@ -22,6 +22,31 @@ You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
 -------------------
+3.18.2 - 2017-08-16
+-------------------
+
+Fix bug where mocks can be used as test runners under certain
+conditions. Specifically, if a mock is injected into a test via pytest
+fixtures or patch decorators, and that mock is the first argument in the
+list, hypothesis will think it represents self and turns the mock
+into a test runner.  If this happens, the affected test always passes
+because the mock is executed instead of the test body. Sometimes, it
+will also fail health checks.
+
+Related to a section of super-bug :issue:`198` and quite possibly
+fixes :issue:`491` (a partial duplicate).
+
+Specifics:
```

Comment:
I'd delete this section - those interested can read the PR discussion, so the changelog can focus on changes to public behaviour.